packages/dds/merge-tree/src/mergeTree.ts

```diff
@@ -2390,7 +2394,7 @@ export class MergeTree {
 	}
 
 	private blockUpdate(block: IMergeBlock) {
-		let len = 0;
+		let len: number | undefined;
```

Comment:
Total nit: I prefer to use length instead of len for the variable name. Probably not worth changing if it's used in code that you're not changing. No need to balloon the change.